exercises/rna-transcription/example/RNA.ceylon

```diff
@@ -0,0 +1,13 @@
+class Error() {}
+
+String|Error transcription(String dna) {
+  return dna.fold("" of String|Error)((String|Error r,
+                                       Character    c) =>   if(is Error r)
```

Comment:
the `if` seems to be indented farther to the right than the `then` and `else`. Is there a reason for that?